### PR TITLE
[ENH] reconfig HC tests

### PR DESCRIPTION
**Pull request by TonyBagnall**
This PR parameterises the default estimator parameters for components of HC1 and HC2 so that we can test default behaviour in reasonable time

### CODE REVIEW

File: aeon/classification/hybrid/_hivecote_v1.py [hunk 1]
```diff
@@ -165,13 +170,16 @@ def _fit(self, X, y):
         """
         # Default values from HC1 paper
```
Comment (MatthewMiddlehurst) [inline aeon/classification/hybrid/_hivecote_v1.py:171]: This is no longer true given the changes

File: aeon/classification/hybrid/_hivecote_v1.py [hunk 2]
```diff
@@ -148,6 +148,11 @@ def __init__(
 
         super(HIVECOTEV1, self).__init__()
 
+    DEFAULT_N_TREES = 500
```
Comment (MatthewMiddlehurst) [inline aeon/classification/hybrid/_hivecote_v1.py:151]: If we implement this it should maybe for tsf and rise. No strong feelings really, this is probably fine

File: aeon/classification/hybrid/_hivecote_v2.py [hunk 3]
```diff
@@ -171,13 +179,20 @@ def _fit(self, X, y):
         """
         # Default values from HC2 paper
```
Comment (MatthewMiddlehurst) [inline aeon/classification/hybrid/_hivecote_v2.py:180]: see above

File: aeon/classification/hybrid/_hivecote_v2.py [hunk 4]
```diff
@@ -149,6 +149,14 @@ def __init__(
 
         super(HIVECOTEV2, self).__init__()
 
+    DEFAULT_N_TREES = 500
+    DEFAULT_N_SHAPELETS = 10000
+    DEFAULT_N_KERNELS = 2000
+    DEFAULT_N_ESTIMATORS = 25
+    DEFAULT_N_PARA_SAMPLES = 250
+    DEFAULT_MAX_ENSEMBLE_SIZE = 50
+    DEFAULT_RAND_PARAMS = 50
```
Comment (MatthewMiddlehurst) [inline aeon/classification/hybrid/_hivecote_v2.py:158]: maybe these should be protected? seems to be only used for testing. dont know how it interacts with docs/idea auto complete etc.

**Review by MatthewMiddlehurst — COMMENTED** [review]
minor changes, feel free to dismiss. If this changes will need to remember to update our group eval package for new defaults when release is updated.

File: aeon/classification/hybrid/_hivecote_v1.py (hunk 1 above)
Comment (TonyBagnall) [reply aeon/classification/hybrid/_hivecote_v1.py]: fixed

File: aeon/classification/hybrid/_hivecote_v2.py (hunk 4 above)
Comment (TonyBagnall) [reply aeon/classification/hybrid/_hivecote_v2.py]: I've made them all protected. I can imagine cases a user might want to change these, but its pretty obscure

**Comment by TonyBagnall** [comment]
> minor changes, feel free to dismiss. If this changes will need to remember to update our group eval package for new defaults when release is updated.

I thought I kept the defaults the same? Which have changed?

File: aeon/classification/hybrid/_hivecote_v1.py (hunk 2 above)
Comment (TonyBagnall) [reply aeon/classification/hybrid/_hivecote_v1.py]: yes agreed, but will look at that when I get round to coverage there

**Review by MatthewMiddlehurst — APPROVED** [review]

**Pull request merged**